include/GL/glui.h

```diff
@@ -581,7 +642,7 @@ class GLUIAPI GLUI_Glut_Window : public GLUI_Node
     void (*glut_mouse_CB)(int,int,int,int);
     void (*glut_visibility_CB)(int);
     void (*glut_motion_CB)(int,int);
-    void (*glut_display_CB)(void);
```

Comment:
You're not the only busy person.  Believe me.  Question is whether you prefer to do your own thing, or collaborate.  I'm happy to work with people, do a bit of the grunt work, share some of the tricks of the trade.  We agree here that void should disappear, so why don't we stop disagreeing about it?  Let's make a plan instead.